### ISSUE TITLE
Create interface between TuLiP and LTLMoP

### ISSUE THREAD

**Issue by slivingston** [tulip-control/tulip-control#38]
(Copied from https://sourceforge.net/p/tulip-control/feature-requests/10/ with creation date 2012-05-30, with update of assigning to @vraman on 2013-09-03.)

The Linear Temporal Logic MissiOn Planning (LTLMoP) toolkit is a software package written at Cornell designed to assist in the rapid development, implementation, and testing of high- level robot controllers. In this toolkit, structured English and Linear Temporal Logic are used to write high-level reactive task specifications, which are then automatically transformed into correct robot controllers that can be used to drive either a simulated or a real robot.

It would be useful to have a way of taking LTLMoP specifications and converting them to a form that is compatible with TuLiP.

**Comment by slivingston** [comment]
To make this more interesting, we should take the action item here to be: subsume LTLMoP. In terms of capabilities, we are not far from being able to do so. Beware that LTLMoP is under GPL, so be careful not to use their code to avoid GPL-license-pollution.

**Comment by murrayrm** [comment]
I agree!  Given that LTLMoP is, in a simplistic sense, a front end for GR(1) synthesis, it totally fits within the vision for TuLiP.  A first step would be to be able to parse LTLMoP input into grspec compatible form.
